lib/landing_page.dart

```diff
@@ -0,0 +1,449 @@
+import 'dart:ui';
+import 'package:flutter/material.dart';
+import 'dart:math';
+import 'package:url_launcher/url_launcher.dart';
+
+class LandingPage extends StatelessWidget {
+  final Function goHome;
+  LandingPage({Key key, this.goHome}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
This is too much, uncommented code in one build method/one file. I cannot review really.